code/modules/core_implant/cruciform/rituals/path.dm

```diff
@@ -719,90 +651,158 @@
 
 // Self-Repair
 /datum/ritual/cruciform/factorial/self_repair
-	name = "Litany of the iron soul"
-	desc = "Use the energy in your cruciform to repair all mechanical parts on the bearer, be they synthetic limbs or organs."
-	phrase = "Sic invocamus Absoluta. Ergo omne quod facimus separabuntur."
+	name = "Litany of the Iron Soul"
+	desc = "Use the energy in your cruciform to repair all mechanical parts on the bearer, be they synthetic limbs or organs. The energy required to do so will knock you down for a few seconds, and lessen your combat capabilities for five minutes."
+	phrase = "Absolutas sustentat me." //"The Absolute sustains me."
 	cooldown = TRUE
 	cooldown_time = 10 MINUTES
 	cooldown_category = "repair"
-	power = 35
-	nutri_cost = 150
-	blood_cost = 80 //Aheal but not AOE, so little less bad
+	power = 50
+
+/datum/ritual/cruciform/factorial/self_repair/perform(mob/living/carbon/human/user, obj/item/implant/core_implant/cruciform/C)
+	var/debuff_amount = -5
```

Comment:
these should be moved to the self_repair path itself for editing via debug/admins